modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/base_contestable_issues_controller.rb

```diff
@@ -35,8 +35,12 @@ def index
 
   attr_reader :caseflow_response, :backend_service_exception
 
+  def request_headers
+    EXPECTED_HEADERS.index_with { |key| request.headers[key] }.compact
```

Comment:
Still feel like this should work:
```suggestion
    request.headers.slice EXPECTED_HEADERS
```

Reply:
This change caused some test failures @ericboehs, I reverted back to the original